clients-src/modules/ROOT/pages/clients.adoc

```diff
@@ -6,125 +6,76 @@
+====
 
 [#_typedb_drivers]
 == TypeDB Drivers
 
-A TypeDB Driver, is a library that implements https://github.com/vaticle/typedb-protocol[TypeDB Client RPC,window=_blank]
-protocol and provides TypeDB Driver API.
+A TypeDB Driver is a library that implements https://github.com/vaticle/typedb-protocol[TypeDB Client RPC,window=_blank]
+protocol. TypeDB Drivers are available for some of the most popular programming languages, allowing you to
+build applications that connect to a TypeDB database easily.
 
-A TypeDB Driver can be used to connect to a TypeDB server. If we are building an application that uses a TypeDB database,
-we would need a TypeDB driver at our application layer to handle the database operations.
-TypeDB Drivers are available for some of the most popular programming languages.
-
-The following TypeDB Drivers are officially supported and actively maintained by the Vaticle. They
-usually support latest TypeDB features and receive continuous bug fixes and improvements.
+The following TypeDB Drivers are officially supported and actively maintained by Vaticle. They
+support the latest TypeDB features and receive continuous bug fixes and improvements.
 
 * xref:java/java-overview.adoc[Java]
 * xref:node-js/node-js-overview.adoc[Node.js]
 * xref:python/python-overview.adoc[Python]
 * Rust (https://github.com/vaticle/typedb-client-rust[Under development,window=_blank])
 
-We also have some community projects for xref:other-languages.adoc[other Languages] and provide support for
-creating a xref:new-driver.adoc[new client].
+There are also xref:other-languages.adoc[community-developed clients]
+(including a https://pypi.org/project/typedb-jupyter/[Jupyter connector,window=_blank])
+and guides for creating xref:new-driver.adoc[new drivers].
 
 === Architecture of a TypeDB Driver
 
-All TypeDB Drivers share a common architecture. A TypeDB Drivers needs a `gRPC` client library to communicate with
-the server.
-
-A TypeDB Client provides a human or machine interface and interacts with TypeDB server, but also it performs query
-validation, local processing, load-balancing (TypeDB Cloud only) and authentication (TypeDB Cloud only).
-
-The following schema is the dependency graph of a typical TypeDB Client.
+All TypeDB Drivers share a common architecture and capabilities. The core components have to do with opening and managing
+connections, sessions, and transactions, as well as submitting queries and processing responses.
 
 image::client-structure.png[Structure of a TypeDB Client]
 
 //#todo Update Cluster to be Enterprise
 //#todo Redraw in new design
 
-Simply put, the main components of any TypeDB Client are the classes
-and methods to establish a connection to a TypeDB database, execute queries and process responses.
-
-TypeDB Clients in the form of standalone applications usually take care of internal processes and provide a
-user-friendly interface for user (graphical or command-line).
```

Comment:
this is in the wrong section - we're going back to talking about 'clients' generally, when we should be getting more specific within the 'Driver Architecture' section!